bids-validator/bids_validator/rules/file_level_rules.json

```diff
@@ -522,5 +522,37 @@
     "tokens": {
       "@@@_pet_ext_@@@": ["blood\\.tsv\\.gz", "blood\\.tsv", "blood\\.json"]
     }
+  },
+
+  "microscopy": {
+    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?microscopy[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)?(?:_chunk-[a-zA-Z0-9]+)?(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(@@@_microscopy_type_@@@)(@@@_microscopy_ext_@@@)$",
+    "tokens": {
+      "@@@_microscopy_type_@@@": [
+        "_tem",
```

Comment:
Interesting... we were planning to use uppercase for the modality suffix but maybe we should go with lowercase. 
MRI uses a mix or upper and lower but everyone else use lower. I'll get back to you on that.

Reply:
We'll keep uppercase for now, better to have consistency across all repos + BEP. If we decide later to change it, we'll do it everywhere at the same time.

Reply:
Ok I will put the suffixes in uppercase for bids-validator and bids-examples, thanks!